src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -83,6 +83,8 @@
 #include <transformations/utils/utils.hpp>
 #include <snippets/pass/collapse_subgraph.hpp>
 #include "ngraph_transformations/snippets_mark_skipped.hpp"
+#include <transformations/op_conversions/convert_roi_align_v9_to_v0.hpp>
+#include <transformations/op_conversions/convert_roi_align_v0_to_v9.hpp>
 
```

Comment:
better change the name v0 and align with name convention of other ops ... ~~ confusing here it is.